DataProcessing/datax-host/src/main/scala/datax/host/SparkJarLoader.scala

```diff
@@ -23,6 +23,7 @@ import scala.collection.mutable.HashMap
 
 object SparkJarLoader {
   val currentJars = new HashMap[String, Long]
+  var fileUrl = ""
```

Comment:
Make fileUrl as return param of function addJarOnDriver and remove it as global variable.

Reply:
Done